jsk_pr2_robot/jsk_pr2_startup/pr2.launch

```diff
@@ -233,4 +233,18 @@
     </rosparam>
   </node>
 
+  <!-- send email by rostopic -->
+  <node name="email_topic" pkg="jsk_robot_startup" type="email_topic.py" output="screen">
+    <rosparam>
+      email_info: /var/lib/robot/email_topic.yaml
+    </rosparam>
+  </node>
+
+  <!-- send email, twitter with smach -->
+  <node name="server_name" pkg="jsk_robot_startup" type="smach_to_mail.py" output="screen">
```

Comment:
name should be more detailed like smach_to_mail_server

Reply:
OK. I'll fix it.

Reply:
I fixed in https://github.com/knorth55/jsk_robot/pull/338/commits/576a8df7977c7c7c0bf5338bfbf818bbdc5bab15